src/service.js

```diff
@@ -6,8 +6,9 @@ let server;
 exports.start = function start(port = 8080) {
   let app = express();
   app.get('/', (req, res) => res.send('200 OK'));
-  app.get('/connect', require('./connect'));
+  app.get('/connect/:port', require('./connect'));
```

Comment:
Instead of creating `GET` connections to `/connect`, how about making the endpoint `/connection/:port`, and making `POST` requests to create the connection, and `DELETE` requests to remove them? Seems a lot more RESTy.